user-guide/Advanced_Functionality/Databases/Indexing_database/Migrating_from_Elasticsearch_to_Opensearch.md

```diff
@@ -0,0 +1,205 @@
+---
+uid: Migrating_from_Elasticsearch_to_OpenSearch
+---
+
+# Migrating from Elasticsearch to OpenSearch
+
+From DataMiner 10.4.4/10.5.0 onwards<!-- RN 37994 -->, a tool is available that allows you to migrate from Elasticsearch 6.8.22 to OpenSearch 2.11.1.
+
+To use this tool, follow the instructions below:
+
+1. [Take a snapshot of the Elasticsearch 6.8.22 cluster](#take-a-snapshot-of-the-elasticsearch-6822-cluster).
+
+1. [Copy the snapshot to an Elasticsearch 7.10.0 cluster and restore it](#restore-the-snapshot-on-an-elasticsearch-7100-cluster).
+
+1. [Run the re-indexing tool and take a snapshot](#run-the-re-indexing-tool-and-take-a-snapshot).
+
+1. [Copy the snapshot with the re-indexed data to an OpenSearch 2.11.1 cluster and restore it](#restore-the-snapshot-with-the-re-indexed-data-to-a-opensearch-2111-cluster).
+
+> [!TIP]
+> See also: [Taking a snapshot of one Elasticsearch cluster and restoring it to another](xref:Taking_snapshot_Elasticsearch_cluster_and_restoring_to_different_cluster)
+
+## Take a snapshot of the Elasticsearch 6.8.22 cluster
+
+Using Kibana, you can take a snapshot in the following way:
+
+1. Check the *path.repo* configuration in *elasticsearch.yml*.
+
+   This configuration should point to a shared file system location to which each node has access.
+
+1. Check the existing repositories by sending the following request:
```

Comment:
What is the purpose of this check? What should the result be?

Reply:
added info. in case the desired snapshot already exists the next steps can be skipped